README.md

```
@@ -7,16 +7,16 @@ https://ara-ta3.github.io/
 
 フロントエンドのコードは `frontend/` ディレクトリ以下に配置されています。pnpm のワークスペース機能を利用して管理しています。
 
-バックエンドは `backend/` に Scala で置く構想がありますが、今のところやりたいことが不明確で未実装です。  
+バックエンドは `backend/` に Scala で配置しています。試験的な HTTP サーバーがあり、フロントエンドとは未連携です。
 
 ## 技術構成図
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)